frontend/src/components/common/form/StepForm/context.tsx

```diff
@@ -39,6 +39,7 @@ export const StepperFormProvider: React.FC<IStepperFormProviderProps> = ({
   };
 
   const gotoTab = (index: number) => {
+    debugger;
```

Comment:
this has been removed in the next commit, so safe to ignore here.

Reply:
oops commented on other one lol got it